src/App.test.js

```diff
@@ -813,6 +821,7 @@ describe('App component', () => {
           gtag: jest.fn(),
           Menubutton: MenubuttonFake,
         },
+        dopplerSitesClient: dopplerSitesClientDouble,
```

Comment:
In these cases where we need the default dependencies with few differences, I think that spread operator could work very well.